.github/workflows/packaging.yaml

```diff
@@ -33,8 +33,8 @@ defaults:
     shell: bash
 
 jobs:
-  msi:
-    name: "Test Windows Installer build (${{ matrix.os }})"
+  packaging:
+    name: "Test packaging build (${{ matrix.os }})"
```

Comment:
Since it is not all about Windows Installer anymore. Is "packaging" here is the correct replacement?

Reply:
yep